Makefile

```diff
@@ -40,6 +40,9 @@ uirunalpha:
 uirunbeta:
 	export API_URL=https://beta-curation.alliancegenome.org; make -B -C src/main/cliapp run; unset API_URL
 
+uirunprod:
+	export API_URL=https://curation.alliancegenome.org; make -B -C src/main/cliapp run; unset API_URL
```

Comment:
@adamgibs given that this would be against production I typically an reluctant to have this in the Makefile (oops i was pointed at prod)

Reply:
Sure, I went ahead and removed it.